lang/c++/api/Validator.hh

```diff
@@ -33,32 +34,32 @@ class AVRO_DECL NullValidator : private boost::noncopyable {
 public:
 
     explicit NullValidator(const ValidSchema &schema) {}
-    NullValidator() {}
+    NullValidator() = default;
 
-    void setCount(int64_t val) {}
+    void setCount(int64_t) {}
 
-    bool typeIsExpected(Type type) const {
+    static bool typeIsExpected(Type) {
```

Comment:
this change seems a little strange to me (while probably technically correct)

Reply:
This function is not used and hence it doesn't matter, as of now.